### PR TITLE
[TW-1455] Update group info not supported in Okta

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Explained that "Update group information" (i.e. group push) isn't supported in the Postman Okta app. Confirmed that this feature is supported in OneLogin and Entra ID.
* Linked to another Okta app that supports group push. Users can use this app instead if they want the group push provisioning feature.

### CODE REVIEW

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ Postman supports the following provisioning features:
 
         > Users in a group have the roles assigned to the group. You can [update group roles in Postman](/docs/collaborating-in-postman/user-groups/#editing-team-roles-for-a-group) and [control a group's access to workspaces and individual Postman elements](/docs/collaborating-in-postman/user-groups/#managing-roles-on-workspaces-and-postman-elements), such as collections and APIs.
 
+    > Updating group information from your IdP to Postman isn't supported in [SCIM provisioning through Okta](http://localhost:8000/docs/administration/scim-provisioning/configuring-scim-with-okta/). If you're using Okta and you want to use this provisioning feature, use the [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/).
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md:81]:
```suggestion
    > Updating group information from your IdP to Postman isn't supported in [SCIM provisioning through Okta](/docs/administration/scim-provisioning/configuring-scim-with-okta/). If you're using Okta and you want to use this provisioning feature, use the [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/).
```

**Review by jkonrath-postman — APPROVED** [review]
LGTM - make sure I got that link right in the suggestion, I didn't test it.

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ updated: 2023-04-12
 
 Postman supports SCIM provisioning through Okta with the [Postman Okta app](https://www.okta.com/integrations/postman/), enabling you to automate user provisioning and de-provisioning for your team.
 
+> Updating group information from your IdP to Postman isn't supported in SCIM provisioning through Okta. If you're using Okta and you want to use this provisioning feature, use the [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/).
```
Comment (PriyambadaMadala) [inline src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md:8]: `Updating group information from your IdP to Postman`

Apologies for confusion in our previous discussion . 
We need to mention `Updating group information from Postman to your IdP`

File: src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md [hunk 3]
```diff
@@ -78,6 +78,8 @@ Postman supports the following provisioning features:
 
         > Users in a group have the roles assigned to the group. You can [update group roles in Postman](/docs/collaborating-in-postman/user-groups/#editing-team-roles-for-a-group) and [control a group's access to workspaces and individual Postman elements](/docs/collaborating-in-postman/user-groups/#managing-roles-on-workspaces-and-postman-elements), such as collections and APIs.
 
+    > Updating group information from your IdP to Postman isn't supported in [SCIM provisioning through Okta](/docs/administration/scim-provisioning/configuring-scim-with-okta/). If you're using Okta and you want to use this provisioning feature, use the [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/).
```
Comment (PriyambadaMadala) [inline src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md:81]: Same as above we need to change the message `Updating group information from Postman to your IdP`

**Review by PriyambadaMadala — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md (hunk 2 above)
Comment (ArjunSingh-PM) [reply src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md:8]: Updating group information from Postman to your IdP via the official [Postman Okta app](https://www.okta.com/integrations/postman/) isn't supported at the moment. We are working with Okta to have the official [Postman Okta app](https://www.okta.com/integrations/postman/) updated to support the same. 

However, you can use the generic [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/) to unblock updating groups from Postman to your IdP as well.

File: src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md (hunk 3 above)
Comment (ArjunSingh-PM) [reply src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md:81]: Updating group information from Postman to your IdP via the official [Postman Okta app](https://www.okta.com/integrations/postman/) isn't supported at the moment. We are working with Okta to have the official [Postman Okta app](https://www.okta.com/integrations/postman/) updated to support the same.

However, you can use the generic [SCIM 2.0 Okta app](https://www.okta.com/integrations/scim-2-0-test-app-header-auth/) to unblock updating groups from Postman to your IdP as well.

**Review by ArjunSingh-PM — CHANGES_REQUESTED** [review]
Thanks for sharing the changes. I have left a comment that needs to be addressed before pushing this change. Thanks.

File: src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/scim-provisioning/configuring-scim-with-okta.md]: This is now updated. Thanks!

File: src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md (hunk 3 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md]: This is now updated. Thanks!

**Review by ArjunSingh-PM — APPROVED** [review]
Looks good!

**Pull request merged**